File: Java_Programs/Java_Lab_Program/Lab_1/Pattern_4.java
//   * * * * *
//    * * * *
//     * * *
//      * *
//       *

import java.util.Scanner;

public class Pattern_4 {
  public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);

        System.out.println("Enter Any Number => ");  
        int user=sc.nextInt();

        for(int i=user; i>=1; i--){
            for(int space=i; space<user; space++){
                System.out.print(" ");
            }
            for(int j=1; j<=i; j++){
                System.out.print("* ");
            }
            System.out.println();
        }

  }    
}

File: Patterns/Space_Pattern/Pattern_4.c
#include<stdio.h>

// 6 5 4 3 2 1 
// - 6 5 4 3 2 
// - - 6 5 4 3 
// - - - 6 5 4 
// - - - - 6 5 
// - - - - - 6

int main(){

    int n;

    printf("Enter N : ");
    scanf("%d",&n);

    for(int i=1; i<=n; i++){
        
        for(int space=1; space<i; space++){
            printf(" ");
        }

        for(int j=n; j>=i; j--){
            printf("%d",j);
        }

        printf("\n");
    }

}